demos/wasmcloud/fileserver/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+use wapc_guest::prelude::*;
+use wasmcloud_actor_blobstore as blobstore;
+use wasmcloud_actor_http_server as http;
+
+#[no_mangle]
+pub fn wapc_init() {
+    wasmcloud_actor_core::Handlers::register_health_request(health);
+    http::Handlers::register_handle_request(fetch);
+}
```

Comment:
This isn't necessary, but we have a derive macro that allows you to skip the health check definition if you're just going to return healthy, it looks like:
```
#[wasmcloud_actor_core::init]
pub fn init() {
  http::Handlers::register_handle_request(fetch);
}
```
All it does is define a health check that returns healthy for you, so you can avoid defining your own function

Reply:
I'm all for shortening (both code and the cooking fat). I'll fix it right now